pose-detection/src/calculators/interfaces/config_interfaces.ts

```diff
@@ -64,6 +64,11 @@ export interface OneEuroFilterConfig {
       number;  // If calculated object scale is less than given value smoothing
   // will be disabled and keypoints will be returned as is. This
   // value helps filter adjust to the distance to camera.
+  disableValueScaling?:
```

Comment:
Added but not used

Reply:
This change was temporary and is now added and used in  Add 3D keypoints output to TFJS blazepose commit